docs/library/mimxrt.Flash.rst

```diff
@@ -0,0 +1,35 @@
+.. currentmodule:: mimxrt
+.. _mimxrt.Flash:
+
+class Flash -- access to built-in flash storage
+===============================================
+
+This class gives access to the SPI flash memory.
+
+In most cases, to store persistent data on the device, you'll want to use a
+higher-level abstraction, for example the filesystem via Python's standard file
+API, but this interface is useful to :ref:`customise the filesystem
+configuration <filesystem>` or implement a low-level storage system for your
+application.
+
+
+Constructors
+------------
+
+.. class:: Flash()
+
+   Gets the singleton object for accessing the SPI flash memory.
+
+
+Methods
+-------
+
+.. method:: Flash.readblocks(block_num, buf)
```

Comment:
As a help for the reader you could tell, that the block_size can be obtained by a call to ioctl(5,0). The block numbers count from the start of the user flash area, not the physical start of flash.

Reply:
Thank you.